Sources/PointFree/Tasks.swift

```diff
@@ -0,0 +1,252 @@
+        episode, and click the "\(useCreditCTA)" button!
+
+        Here are some of the top episodes that viewers have chosen to use their credits on:
+
+        * [Dependency Injection Made Easy](https://www.pointfree.co/episodes/ep16-dependency-injection-made-easy)
+        * [Algebraic Data Type: Part 1](https://www.pointfree.co/episodes/ep4-algebraic-data-types)
+        * [Tagged](https://www.pointfree.co/episodes/ep12-tagged)
+        * [Contravariance](https://www.pointfree.co/episodes/ep14-contravariance)
+        """
+        )
+      : nil
+    ,
+    markdownBlock(
+      """
+      When you're ready to subscribe for yourself _or_ your team, visit
+      [our subscribe page](\(url(to: .pricing(nil, expand: nil))))!
+      """
+    ),
+    subscribeButton,
+    ]
+    .compactMap(id)
+    <> hostSignOffView.view(unit)
+}
+
+func welcomeEmail2(_ user: Database.User) -> Email {
+  let subject = "Free episodes on Point-Free"
+  return prepareWelcomeEmail(
+    to: user,
+    subject: subject,
+    content: welcomeEmailView(subject, welcomeEmail2Content)
+  )
+}
+
+let welcomeEmail2Content = View<Database.User> { user -> [Node] in
+  let freeEpisodeLinks = Current.episodes()
+    .filter { !$0.subscriberOnly }
+    .map {
+      """
+      * [\($0.title)](\(url(to: .episode(.left($0.slug)))))
+      """
+    }
+    .joined(separator: "\n")
+  return [
+    markdownBlock(
+      """
+      👋 Hey there!
+
+      You signed up for a [Point-Free](\(url(to: .home))) account a couple weeks ago but still haven't subscribed!
+
+      If you're still on the fence and want to see a little more of what we have to offer, we have a number
+      of free episodes for you to check out!
+
+      \(freeEpisodeLinks)
+      """
+    ),
+    user.episodeCreditCount > 0
+      ? markdownBlock(
+        """
+        You *also* have a **free episode credit** you can use to see *any* _subscriber-only_ episode,
+        completely for free! Just visit [our site](\(url(to: .home))), go to an episode, and click the "\(useCreditCTA)" button.
+        """
+        )
+      : nil
+    ,
+    markdownBlock(
+      """
+      If you have any questions, don't hesitate to reply to this email!
+
+      When you're ready to subscribe for yourself _or_ your team, visit
+      [our subscribe page](\(url(to: .pricing(nil, expand: nil))))!
+      """
+    ),
+    subscribeButton,
+    ]
+    .compactMap(id)
+    <> hostSignOffView.view(unit)
+}
+
+func welcomeEmail3(_ user: Database.User) -> Email {
+  let subject = "Here's a free episode!"
+  return prepareWelcomeEmail(
+    to: user,
+    subject: subject,
+    content: welcomeEmailView(subject, welcomeEmail3Content)
+  )
+}
+
+let welcomeEmail3Content = View<Database.User> { user -> [Node] in
+  [
+    markdownBlock(
+      """
+      👋 Hiya!
+
+      We just wanted to reach out one last time in the hope that we might make a subscriber out of you yet,
+      so we've given you another **free episode credit**.
```

Comment:
oh, do we actually give them this credit now? I couldn't find that code...

Reply:
Oh yeah. Lemme TODO it before this gets merged.

Reply:
Added logic!